ui_framework/doc_site/src/views/flex/flex_justify.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import {
+  KuiFlexGroup,
+  KuiFlexItem,
+} from '../../../../components';
+
+export default () => (
+  <div>
+    <KuiFlexGroup growItems={false} justifyContent="spaceBetween">
+      <KuiFlexItem>One here on the left</KuiFlexItem>
+      <KuiFlexItem>The other over here on the right.</KuiFlexItem>
+    </KuiFlexGroup>
+
+    <br/><br/>
+
+    <KuiFlexGroup growItems={false} justifyContent="spaceAround">
+      <KuiFlexItem>I'm a single centered item!</KuiFlexItem>
```

Comment:
Could you run `npm i`? I think you might have an older linter installed. The linter should complain about this apostrophe and request you use an HTML entity instead (`&rsquo;`).